luamocks/autogenerated/SyncedCtrl.lua

```diff
@@ -8,6 +8,15 @@
 ---@return nil
 function Spring.SetAlly(firstAllyTeamID, secondAllyTeamID, ally) end
 
+---Changes the start box position of an allyTeam.
```

Comment:
Other functions say "Parameters" here?

Reply:
Yes: the reason is copy/paste 😛 This first item is supposed to be the description of the thing, and you can check if hover anything in VS Code that all the others will say "parameters" in IntelliSense because of that, haha. Here is the full spec for Lua type annotations => https://github.com/LuaLS/lua-language-server/wiki/Annotations